model/minio/minio_cluster.rb

```diff
@@ -60,4 +67,29 @@ def single_instance_multi_drive?
   def hostname
     "#{name}.#{Config.minio_host_name}"
   end
+
+  def certs
```

Comment:
root_certs instead of certs maybe? certs can refer server/client certs as well.

Reply:
refactored as requested.